styles/main.css

```diff
@@ -215,7 +215,7 @@ img {
         display:none;
     }
     .mugshot {
-        clear:right;
+        float: none;

```

Comment:
You need to remove the floatiness of the element, not make it clear other floats